lib/api_repo.js

```diff
@@ -463,6 +519,38 @@ ApiRepo.prototype._buildProtos =
       }
     };
 
+/**
+ * Collect the required files from a .proto file. 'done' will be
+ * called with the list of file name within the includePath when
+ * the task has finished.
+ */
+function collectProtoDeps(protoFile, includePath, done) {
```

Comment:
Good catch. Syntactically this works as expected, but for the style/idioms, I think to bring prior to the user function. Changed like that.